static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -179,7 +179,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.FIRST_INTEGRATION,
       title: t('Install any of our 40+ integrations'),
       description: t(
-        'Get alerted in Slack. Two-way sync issues between Sentry and Jira. Notify Sentry of releases from GitHub, Vercel, or Netlify.'
+        'Get alerts in Slack, sync Sentry and Jira, notify Sentry of releases from GitHub, Vercel, or Netlify.'
```

Comment:
please keep the original. two-way sync is important.

Reply:
ok.

Reply:
hm.. maybe. the sync is between sentry issues and jira issues. that's why it's important to call out. i'm not sure we gain much by changing it.

Reply:
For reference, this is how it looks in-app today:

<img width="423" alt="Screen Shot 2022-11-15 at 11 45 34 AM" src="https://user-images.githubusercontent.com/187460/202011445-35970a04-367e-44e6-a7eb-6821a7b292db.png">

Reply:
I don't want to block on this point, so i'll also add here a suggestion to fully revert. So we can revert by clicking the button here, or resolve the conversation with the change to a single sentence.

```suggestion
        'Get alerted in Slack. Two-way sync issues between Sentry and Jira. Notify Sentry of releases from GitHub, Vercel, or Netlify.'
```